### PR TITLE
[ci] [appveyor] Disable validate target.

### PR DESCRIPTION
**Pull request by ejgallego**
Not a big point on running the checker on Appveyor I think, this will
save a bunch of time and improve reliability.

### CODE REVIEW

File: dev/ci/appveyor.sh [hunk 1]
```diff
@@ -12,4 +12,4 @@ opam init -a mingw https://github.com/fdopen/opam-repository-mingw.git --comp $A
 eval "$(opam config env)"
 opam install -y num ocamlfind camlp5 ounit
 
-cd "$APPVEYOR_BUILD_FOLDER" && ./configure -local && make && make byte && make -C test-suite all INTERACTIVE= && make validate
+cd "$APPVEYOR_BUILD_FOLDER" && ./configure -local && make && make byte && make -C test-suite all INTERACTIVE= # && make validate
```
[the remark this answers is not in the document]
Comment (ejgallego) [reply dev/ci/appveyor.sh]: It may prove useful if someone migrates this script to Travis or Gitlab.

**Comment by SkySkimmer** [comment]
That's nonsense, adding make validate is no effort.

Gaëtan Gilbert

**Comment by ejgallego** [comment]
> That's nonsense, adding make validate is no effort.

You may remember that this target used to do validate, but the person coming after you certainly won't.

**Comment by Zimmi48** [comment]
It makes sense to keep the `validate` commented. In fact, it would be even better to try to port this to Travis or Azure pipelines (I would need to look into this further, but I think I have a preference for a CI that supports pipeline, i.e. artifact reuse, out of the box, not like Travis).

**Comment by ejgallego** [comment]
We are hitting again quite a few timeouts on appveyor, can we merge this @maximedenes ?

I prefer to keep the comment, as a reminder to re-enable this when it is migrated

**Comment by ejgallego** [comment]
Ping.

**Comment by ejgallego** [comment]
@SkySkimmer can you take this one? Merci.

**Comment by maximedenes** [comment]
I can do it, sorry for the delay, struggling against flu.

**Review by maximedenes — APPROVED** [review]

**Comment by ejgallego** [comment]
> I can do it, sorry for the delay, struggling against flu.

Thanks! It is just a very small nit but it will help clearing the current backlog. Caching could be interesting too c.f. #8711

**Pull request merged**